doc/tutorials/dune-package-management/repos.md

```diff
@@ -0,0 +1,49 @@
+## Custom Repositories
+
+In normal usage Dune looks up packages from two sources:
+
+  1. The upstream, community maintained opam repository at
```

Comment:
If this is supposed to be `opam-repository`, I will go through and change them all.

Reply:
I replaced it with "package repository" given we link to `opam-repository` right afterwards, that way it should be less repetitive while minimizing confusion (I hope).